app/assets/stylesheets/application.css

```diff
@@ -17,3 +17,19 @@
  @import "tailwindcss/base";
  @import "tailwindcss/components";
  @import "tailwindcss/utilities";
+
+h1 {
+    @apply text-4xl md:text-5xl xl:text-6xl;
```

Comment:
I think for CSS is also suggested to use 1 tab(two spaces) in the identation. It's just a sugestion, if in the project you all want to work with two tabs, it's fine. Just try to make it standard.

Reply:
I'll check with my mates and decide which one to use, thank for the suggestion!